packages/scandipwa/src/component/ProductCustomizableOption/ProductCustomizableOption.container.js

```diff
@@ -174,26 +187,22 @@ export class ProductCustomizableOptionContainer extends PureComponent {
     }
 
     getDropdownOptions(values) {
-        return values.reduce((acc, {
+        return values.map(({
             option_type_id,
             title,
             priceInclTax,
             price,
             price_type,
             sort_order,
             currency
-        }) => {
-            acc.push({
-                id: option_type_id,
-                name: title,
-                value: option_type_id,
-                label: `${title} `,
-                subLabel: this.renderOptionLabel(price_type, priceInclTax, price, currency),
-                sort_order
-            });
-
-            return acc;
-        }, []);
+        }) => ({
+            id: option_type_id,
+            name: title,
+            value: option_type_id,
+            label: `${title} `,
```

Comment:
Is it necessary to add a space there? I believe it would be better to add a padding or margin than space in the data object.